benches/gdelt/main.rs

```diff
@@ -0,0 +1,261 @@
+#![feature(closure_lifetime_binder)]
```

Comment:
Forgot to remove that, I tried to make some of our operators (`.flat_map()` specifically) not require returning owned iterators (meaning that we could just return `slice.iter().cloned()` instead of having to clone the entire vec just for it to not be used) but it was enough work that it deserves its own PR